test/test_cpp_extensions_mtia_backend.py

```diff
@@ -24,7 +25,13 @@
 
 
 @unittest.skipIf(
-    IS_ARM64 or not IS_LINUX or TEST_CUDA or TEST_PRIVATEUSE1 or TEST_ROCM or TEST_XPU,
+    IS_ARM64
+    or not IS_LINUX
+    or TEST_CUDA
+    or TEST_PRIVATEUSE1
+    or TEST_ROCM
+    or TEST_XPU
+    or TEST_MPS,
```

Comment:
Otherwise, UT `test_device_context` will raise an Error `Only one device type can be registered. But now, we have two device types: mps and mtia` on mps CI.